two_encoder_module/two_encoder_src/two_encoder_src.ino

```diff
@@ -109,18 +115,17 @@ struct Encoder
     uint8_t pinB;
     uint8_t buttonPin;
     uint8_t buttonState;
-    int pinAPrevious;
-    int pinBPrevious;
+    uint8_t pinAPrevious;
+    uint8_t pinBPrevious;
```

Comment:
pinAPrevious and pinBPrevious should be changed back to int, see comment about digitalRead returns below.